libzeth/tests/core/ec_operation_data_test.cpp

```diff
@@ -15,6 +15,30 @@ using namespace libzeth;
 namespace
 {
 
+// Utility function to compute a point on the curve E(Fq) with the given x
+// coordinate.
+template<typename GroupT>
+static GroupT g1_curve_point_at_x(const typename GroupT::base_field &x)
+{
+    const typename GroupT::base_field x_squared = x * x;
+    const typename GroupT::base_field x_cubed = x_squared * x;
+    const typename GroupT::base_field y_squared =
+        x_cubed + (GroupT::coeff_a * x_squared) + GroupT::coeff_b;
+    const typename GroupT::base_field y = y_squared.sqrt();
+    return GroupT(x, y, GroupT::base_field::one());
```

Comment:
See: https://github.com/clearmatics/libff/pull/31/files#r543460055 - let's check that a root can be extracted (i.e. compute Legendre symbol) before calling `sqrt`

Reply:
Thanks.  If the util functions are merged (see https://github.com/clearmatics/libff/pull/31#discussion_r544923921) we can change this to reference them instead.

Reply:
Replaced by util fuctions in libff